tests/test_use_index_title.py

```diff
@@ -0,0 +1,210 @@
+def test_use_index_title_disabled_by_default(mkdocs):
```

Comment:
Could you drop `use_index_title_` from all the function titles? It's a bit redundant since it's already in the filename.

Reply:
Refactored